src/nre.nim

```diff
@@ -427,7 +416,9 @@ proc initRegex(pattern: string, options: string): Regex =
 
   result.captureNameToId = result.getNameToNumberTable()
 
-proc re*(pattern: string, options = ""): Regex = initRegex(pattern, options)
+proc re*(pattern: string): Regex =
+  let (pattern, flags, study) = extractOptions(pattern)
```

Comment:
Sounds like I should set up the tester to test with 0.11.0, not just with devel. I'll look into it later today once I get to my PC.